test/sys/test_wait.rs

```diff
@@ -37,6 +37,19 @@ fn test_wait_exit() {
     }
 }
 
+#[test]
+fn test_waitstatus_pid() {
+    let _m = ::FORK_MTX.lock().expect("Mutex got poisoned by another test");
+
+    match fork().unwrap() {
+        Child => {},
```

Comment:
You mean the reasons like this: https://stackoverflow.com/questions/5422831/what-is-the-difference-between-using-exit-exit-in-a-conventional-linux-fo
(double buffer flushing or double execution of `atexit` handlers?)